GoogleDataTransportCCTSupport/GoogleDataTransportCCTSupport/ProtoSupport/generate_cct_protos.sh

```diff
@@ -0,0 +1,62 @@
+#!/bin/bash
+
+# Copyright 2019 Google
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+# Example usage:
+# ./build_protos <path to nanopb>
+
+# Dependencies: git, protobuf, python-protobuf, pyinstaller
+
+# From https://stackoverflow.com/a/246128/361918
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+
+# Current release of nanopb being used  to build the CCT protos
+NANOPB_ZIPFILE="https://github.com/nanopb/nanopb/archive/nanopb-0.3.9.2.tar.gz"
+
+NANOPB_TEMPDIR="$DIR/nanopb_temp"
```

Comment:
> (Only reviewed the first commit so far)

Sorry for making the PR so enormous! A lot of it was scripts lifted from other places, so that adds to the size. I'm mostly interested in you looking at this commit, and my usage of nanopb.